java/ci/build-in-docker.sh

```diff
@@ -54,16 +53,21 @@ export LIBCUDF_KERNEL_CACHE_PATH=/rapids
 rm -rf "$WORKSPACE/cpp/build"
 mkdir -p "$WORKSPACE/cpp/build"
 cd "$WORKSPACE/cpp/build"
-cmake .. -DUSE_NVTX=$ENABLE_NVTX \
+cmake .. -GNinja \
```

Comment:
consider making it a param with `-GNinja` being default

Reply:
But why?

Reply:
why replace one hardcoding with another instead of offering alternatives for flexibility?

Reply:
Done.